sparkpost.md

```diff
@@ -1,28 +1,29 @@
 # SparkPost
 
-Only needed when you need to send mails.
 
-> **IMPORTANT:** Never ever set up a *renuoapp.ch* as a sending domain to a new created account. **This will block the account!**. If you need this, chose second option.
+:warning:
+**Never ever set up a *\*.renuoapp.ch* as a sending domain to a new created account. This will block the account on SparkPost.
+If you need this, choose the second option.** :warning:
 
 There are two options to set up SparkPost:
 
 * With a domain for the project and a separate SparkPost account (preferred, if possible)
 * Without an exisiting domain ==> via renuoapp.ch SparkPost account
 
-**Note:** With the second option, you are only able to send mails with **####@renuoapp.ch** as your mail sender
+**Note:** With the second option, you are only able to send mails with **\*@renuoapp.ch** as your mail sender.
 
 
 ### First Option
 
-*There should only be one sparkpost account per project. That means, that the master, develop & testing all use the same one.*
+There should only be one SparkPost account per project. This means, that *master*, *develop* and *testing* use the same one.
```

Comment:
This is against what i wrote above:
. As soon as it is live, you will have two: the <project-name>.<domain> will be one and renuoapp has to be on the existing renuo-app as a subaccount.